### DISCUSSION TITLE
请问发生色度溢出，应该如何处理

### DISCUSSION THREAD

**Discussion by liwenxiang1292** [hooke007/mpv_PlayKit#212] [mpv 通用]
发生了vcb那篇常见瑕疵中说明过的“色度溢出（chroma bleeding）”问题，自己尝试加了个KrigBilateral.glsl，好像没有解决，有什么好的处理办法吗？

**Comment by hooke007** [comment]
krig是做YUV-RGB转换用的，没有修复色度问题的功能。
色度溢出一般在转换前就存在了。
大部分问题不能靠套个着色器就修复。
那篇文章的目的是拓展，让你了解乱套着色器/滥用滤镜可能产生/源本身就存在的问题而不是只用着色器就能修复那些问题，不然还要手动的vs压制干啥呢。
